test/run-drun/actor-class-mgmt.mo

```diff
@@ -107,37 +98,17 @@ actor a {
       assert (c3 != c2);
 
       // no need to add cycles
-      // upgrade by using enhanced orthogonal persistence
       let c4 = await
         (system Cs.C)(#upgrade c3)(4, null);
       assert ({args = 4; upgrades = 1} == (await c4.observe()));
       assert (c4 == c3);
 
-      // upgrade by using graph-copy-based stabilization
-      await useIncrementalStabilization(c4).__motoko_stabilize_before_upgrade();
+      // no need to add cycles
       let c5 = await
-        (system Cs.C)(#upgrade c4)(5, null);
-      await useIncrementalStabilization(c5).__motoko_destabilize_after_upgrade();
-      assert ({args = 5; upgrades = 2} == (await c5.observe()));
+        (system Cs.C)(#reinstall c4)(5, null);
+      assert ({args = 5; upgrades = 0} == (await c5.observe()));
       assert (c5 == c4);
 
-      let c6 = await
-        (system Cs.C)(#upgrade_with_persistence { wasm_memory_persistence = #Keep ; canister = c5 })(6, null);
```

Comment:
Is there no more need for #upgrade_with_persistence? Is just subsumed by #upgrade?

Reply:
This is only test restructuring because of the integration: I removed this part from this test file as it runs on both classical persistence and EOP. There is another test `actor-class-mgmt-enhanced.mo` that specifically runs for EOP and contains the upgrade with the explicit persistence mode.